src/grpc/dp_grpc_impl.c

```diff
@@ -485,6 +495,7 @@ static int dp_process_create_interface(struct dp_grpc_responder *responder)
 	rte_memcpy(port->iface.ul_ipv6, ul_addr6, sizeof(port->iface.ul_ipv6));
 	port->iface.cfg.own_ip = request->ip4_addr;
 	port->iface.cfg.ip_depth = DP_LPM_DHCP_IP_DEPTH;
+	port->iface.is_ipv6_set = !dp_is_ipv6_addr_zero(request->ip6_addr);
```

Comment:
Now seeing this code, we could actually maybe just drop the whole `is_ipv6_set` and use the function dorectly as the optimized version is two CMPs?

Reply:
I liked the idea of optimizing the dp_is_ipv6_zero the way you suggested. I think two CMPs also should be good enough, as the loop checking the bool value will be on the "control" thread side.